### DISCUSSION TITLE
Control pybricks hub from smartphone

### DISCUSSION THREAD

**Discussion by AnToHu0** [pybricks/support#1673] [Q&A]
Hi guys! Is there any possibility to use my smartphone as a controller for MOC powered by pybricks? I want to build a car with some extra functions which not presented in any control+ profile so i need to manage a remote somehow.

**Comment by laurensvalk** [comment]
While a bit more limited, you can indeed use your computer or phone keyboard for additional input. See this example: https://pybricks.com/project/pybricks-3-5-released/#new-block-for-keyboard-input
